content/en/docs/Responses/transcodeDecision.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "transcodeDecisionResponse"
+linkTitle: "transcodeDecisionResponse [OS]"
+opensubsonic:
+- Extension
+description: >
+  The response from the getTranscodeDecision endpoint.
```

Comment:
Maybe we should also write a note about validity duration of this response.
Either we say it is server specific, either we add an optional validy duration in the response itself.